EDI/Operation/02_ManageHeaderFileSett.md

```diff
@@ -0,0 +1,404 @@
+
+
+
+## Prepare line item export
+
+In order to be able to export the individual line items of an invoice with specific data sources later, the relevant line item group of an EDIFACT message must be specified to be exported as a whole. You have to do this here in the header definition file settings.  
+After you have prepared the line item export, you can define the definition for the line items, see [Manage definitions](./01_ManageDefinitions.md).
+
+This procedure shows how you must prepare the line item section of the message, so that the *DataHub Exporter* can add all line items to an invoice message later.
+
+#### Prerequisites
+
+- You have expanded the message structure, see [Get header file segments](#get-header-file-segments).
+
+#### Procedure
+
+*DataHub Exporter > EXPORTS tab > Select message definition > File settings tab > Expand message structure*
+
+![Message structure](../../Assets/Screenshots/EDI/Operation/ExportDefMessDefDates.png "[Message structure]")
+
+
+1. Check the EDIFACT message structure, in which group the LIN segment is available. In newer versions of the INVOIC, the LIN segment is included in group 27, in older versions it is in group 26.
+
+2. Scroll down in the message structure until you have found the relevant group, *G_INVOIC_07A_SG26* or *G_INVOIC_13B_SG27*.
+
+3. Click the ![Add XML attribute](../../Assets/Icons/Plus08.png "[Add XML attribute]") (Add XML attribute) button to the right of the *G_INVOIC_07A_SG26* resp. *G_INVOIC_13B_SG27* row. 
+    - The segment is highlighted with a small black border. You cannot expand it.      
+    - Another *G_INVOIC_07A_SG26* or *G_INVOIC_13B_SG27* row has been added. You can ignore it.      
+    - The EDI Export SG26/SG27 *Destination attribute* has been marked for a later data mapping.
+
+4. Click the [SAVE] button.    
+   The *Exporter definitions* view is displayed.
+
+5. If desired, check if the group has been really marked for a later data mapping. For detailed information, see [Prepare data mapping for the line items](./04_ManageLineItemsFileStt.md#prepare-data-mapping-for-the-line-items).   
+    The EDI Export SG[Group name] *Destination attribute* has been added to the attributes to be mapped later or directly afterwards, if desired.
+
+     ![SG26 Mapping](../../Assets/Screenshots/EDI/Operation/ExportDefMapSG26Segment.png "[SG26 Mapping]")
+
+6. Finalize the *Map attributes* wizard by clicking the [CONTINUE] buttons until you can click the [FINALIZE] button.
+  The *Exporter definition sources* view is displayed.
+
+7. Click the [SAVE] button to leave the header definition.   
+   The *Exporter definition* view is displayed.
+
+8. Create the definitions for the line items, see [Create basic definition settings](./01_ManageDefinitions.md#create-basic-definition-settings).
```

Comment:
In Create basic definitions settings, ist ein Prerequisite wie folgt:
Before creating the definition for the line items, you have prepared the line item export, see [Prepare line item export](https://github.com/Actindo-AG/Documentation/blob/ACD-947/EDI/Operation/02_ManageHeaderFileSett.md#prepare-line-item-export).

Generell: Ich finde, die Prozeduren überschneiden sich ein wenig. Die Prerequisites bringen manchmal den Leser, mindestens gefühlt, in einem Kreis zu bewegen. Vielleicht eher eine "geradelinige" Prozedur anbieten, nicht so viele Alternative (z.B. "you can do it later, if desired")...

Reply:
Es handelt sich ja um ein Beipiel.....